server/routes/auth/auth.route.js

```diff
@@ -76,7 +76,7 @@ router.post('/login', (req, res) => {
       const payload = { id: user.id };
       const token = user.signJWT(payload);
       const options = {
-        httpOnly: true,
+        httpOnly: false,
```

Comment:
I think it may be better/safer to transport `jwt` with the `httpOnly` flag. According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies), having it there will make the cookie inaccessible by doing `Document.cookie` JavaScript API

Reply:
Yeah, did that for debugging. Have to change it though. Will do that.